st2tests/requirements.txt

```diff
@@ -7,6 +7,7 @@
 # update the component requirements.txt
 RandomWords
 mock==2.0.0
+more-itertools<=5.0.0; python_version < "3"
```

Comment:
IIRC, for that notation, a more recent version of distutils / setuptools needs to be available during package create and install time, right?